tools/tf_runner.sh

```diff
@@ -33,124 +32,244 @@ function ReadArguments() {
 				shift
 				export COMMAND="destroy"
 				;;
+			-v|--verbose)
+				export VERBOSE=true
+				shift
+				;;
 			*)
-				echo "Unknown option: $1. Use -h for help."
+				echo "ERROR: Unknown option: $1. Use -h for help."
 				exit 1
 				;;
 		esac
 	done
 
 	# Check mandatory parameters
-	if [ ! $COMMAND ]; then
-		echo "Error! --command argument must be specified. Use -h for help!"
+	if [[ ! $COMMAND ]]; then
+		echo "ERROR: Command argument must be specified. Use -h for help!"
 		exit 1
 	fi
-	if [ ! $PROFILE ]; then
-		echo "Error: No PROFILE found. Pass path to the Cluster profile json file as PROFILE env var!"
+	# Validate profile path
+	if [[ ! $PROFILE ]]; then
+		echo "ERROR: No PROFILE found. Pass path to the Cluster profile json file as PROFILE env var!"
 		exit 1
 	fi
 	# Validate Command parameter
 	if [[ ! " ${TF_SUPPORTED_COMMANDS[@]} " =~ " ${COMMAND} " ]]; then
-		echo "Error: incorrect Command parameter \"$COMMAND\", must be one of ${TF_SUPPORTED_COMMANDS[@]}!"
+		echo "ERROR: incorrect Command parameter \"$COMMAND\", must be one of ${TF_SUPPORTED_COMMANDS[@]}!"
 		exit 1
 	fi
+	if [[ $VERBOSE == true ]]; then
+		set -x
+	fi
 }
 
 # Get parameter from cluster profile
 function GetParam() {
 	result=$(jq ".$1" $PROFILE | tr -d '"')
 	if [[ $result == null ]]; then
-		echo "Error: $1 parameter missed in $PROFILE cluster profile"
+		echo "ERROR: $1 parameter is missing in $PROFILE cluster profile"
 		exit 1
 	else
 		echo $result
 	fi
 }
 
+function IngressTFCrutch() {
+	for file in $(find $1 -type f -name "*.tf" ! \( \
+	-name "main.tf" -o \
+	-name "versions.tf" -o \
+	-name "variables.tf" -o \
+	-name "$(echo $(GetParam 'cluster_type') | awk -F\/ '{print $1}').tf" \)); do
```

Comment:
`echo $(GetParam 'cluster_type')` -> `GetParam 'cluster_type'`

Reply:
Fixed in 6d71952